src/server/document-controller.js

```diff
@@ -217,12 +204,14 @@ class DocumentController {
 
     router.get(
       '/docs/:docKey',
+      validateParams(getDocumentParamsSchema),
       (req, res) => this.handleGetDocPage(req, res)
     );
 
     router.get(
-      '/docs/:docKey/*',
-      (req, res) => this.handleGetDocPageWithSlug(req, res)
+      '/docs/:docKey/:docSlug',
```

Comment:
To match also slugs that contain slashes ("catchall"), the route must be:

~~~
'/docs/:docKey/:docSlug*'
~~~

Reply:
See https://github.com/educandu/educandu/pull/146#discussion_r781997236